app/models/user.rb

```diff
@@ -104,6 +105,7 @@ class User < ApplicationRecord
   has_many :post_likes, dependent: :destroy
   has_many :review_likes, dependent: :destroy
   has_many :list_imports, dependent: :destroy
+  has_many :quotes, dependent: :destroy
```

Comment:
It would probably make more sense to use `dependent: :nullify` and remove the non-null constraint on `user_id`

If we add user deletion, it makes more sense to disassociate the quotes from a user than to remove the quotes entirely